### PR TITLE
Added option to control how long the screensaver takes to start

### PR DESCRIPTION
**Pull request by ethanarns**
**Changes**
- Added setting in Display Settings for time until screensaver, defaulting to 3 minutes
- Load the setting from Display Settings to change the time it takes for the screensaver to activate
- Reduced screensaver check time from 10 seconds to 5 seconds
- Added English translations for new settings

**Issues**

### CODE REVIEW

File: src/scripts/screensavermanager.js [hunk 1]
```diff
@@ -11,7 +11,13 @@ import './screensavermanager.scss';
...
     // Returns the minimum amount of idle time required before the screen saver can be displayed
     //time units used Millisecond
-    return 180000;
+    const timeSeconds = userSettings.get('screensaverTime', false);
+    // If it is zero or undefined
+    if (!timeSeconds) {
+        // Defaults to 3 minutes
+        return 180000;
+    }
+    return timeSeconds * 1000;
```
Comment (dmitrylyzo) [inline src/scripts/screensavermanager.js:20]: `screensaverTime` performs the necessary checks.
```suggestion
    return userSettings.screensaverTime() * 1000;
```

File: src/strings/en-gb.json [hunk 2]
```diff
@@ -1769,6 +1769,8 @@
     "ListView": "List View",
     "LabelBackdropScreensaverInterval": "Backdrop Screensaver Interval",
     "LabelBackdropScreensaverIntervalHelp": "The time in seconds between different backdrops when using the backdrop screensaver.",
+    "LabelScreensaverTimeHelp": "The amount of time in seconds of inactivity required to start the screensaver.",
```
Comment (dmitrylyzo) [inline src/strings/en-gb.json:1772]: Don't add strings to the languages other than `en-us`. Only Weblate should be used to work with them.

File: src/strings/en-us.json [hunk 3]
```diff
@@ -599,6 +599,8 @@
     "LabelBaseUrlHelp": "Add a custom subdirectory to the server URL. For example: <code>http://example.com/<b>&lt;baseurl&gt;</b></code>",
     "LabelBackdropScreensaverInterval": "Backdrop Screensaver Interval",
     "LabelBackdropScreensaverIntervalHelp": "The time in seconds between different backdrops when using the backdrop screensaver.",
+    "LabelScreensaverTimeHelp": "The amount of time in seconds of inactivity required to start the screensaver.",
+    "LabelScreensaverTime": "Screensaver Time",
```
Comment (dmitrylyzo) [inline src/strings/en-us.json:603]: Add new strings in alphabetical order (move below and swap).

Comment (ethanarns) [reply src/strings/en-us.json]: Changes applied, thank you

Comment (dmitrylyzo) [reply src/strings/en-us.json]: Also move them below. They should be after: https://github.com/jellyfin/jellyfin-web/blob/b7995eee4a9b13bb67d4adce03da726f22621f6f/src/strings/en-us.json#L865

Comment (ethanarns) [reply src/strings/en-us.json]: Fixed that as well

File: src/scripts/screensavermanager.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ import './screensavermanager.scss';
 function getMinIdleTime() {
     // Returns the minimum amount of idle time required before the screen saver can be displayed
     //time units used Millisecond
-    return 180000;
+    return userSettings.get('screensaverTime') * 1000;
```
Comment (dmitrylyzo) [inline src/scripts/screensavermanager.js:14]: The suggestion from https://github.com/jellyfin/jellyfin-web/pull/6165#discussion_r1788527870 was not fully applied.
```suggestion
    return userSettings.screensaverTime() * 1000;
```

Comment (ethanarns) [reply src/scripts/screensavermanager.js]: My bad, change made

**Pull request merged**